LICENSE.md

```diff
@@ -1,6 +1,6 @@
 # MIT License
 
-Copyright (c) 2021 RStudio, PBC
+Copyright (c) 2021 Posit Software, PBC
```

Comment:
Does the date need to change with the company name? (same question applies to both LICENSE and LICENSE.md)

Reply:
yeah probably should, since Posit didn't exist in 2021...